sanity/sanity_test.go

```diff
@@ -354,3 +354,21 @@ type fakeFsMaker struct{}
 func (fm *fakeFsMaker) Make(device string, fsType string) error {
 	return nil
 }
+
+type fakeResizer struct {
+	resizedMap map[string]struct{}
+}
+
+func (r *fakeResizer) Resize(devicePath, deviceMountPath string) (bool, error) {
+	if r.resizedMap == nil {
+		r.resizedMap = map[string]struct{}{}
+	}
+	key := fmt.Sprintf("%s:%s", devicePath, deviceMountPath)
+	r.resizedMap[key] = struct{}{}
+
+	return true, nil
+}
```

Comment:
@awels not completely sure if the sanity suite does something with this? I saw a similar impl. in the `Mount()` call

Reply:
For the mount call if it calls `Unmount` or `List`, then saving the value in the `Mount` call makes sense so we have something to return. For resize it either succeeds or not. And the sanity suite is only interested in the successful calls I believe, so no real need to save the keys of the paths we are 'resizing'

Reply:
ah okay gotcha

Reply:
No harm done if we do save it, I don't think it will ever be used for anything.